codelabs/gRPC_Cpp_OpenTelemetry_Plugin/completed/greeter_callback_client.cc

```diff
@@ -0,0 +1,156 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
```

Comment:
```suggestion
 * Copyright 2024 gRPC authors.
```

Reply:
no, we're not supposed to change the years since i'm reusing content